src/entities/DealTokenSwap.ts

```diff
@@ -259,6 +259,48 @@ export class DealTokenSwap implements IDeal {
     return this.dealDocument.votingSummary.primaryDAO.votes.concat(this.dealDocument.votingSummary.partnerDAO?.votes ?? []);
   }
 
+  @computedFrom("dealDocument.votingSummary.primaryDAO.votes")
+  public get primaryDAOVotes(): Array<IVoteInfo> {
+    return this.dealDocument.votingSummary.primaryDAO.votes;
+  }
+
+  @computedFrom("dealDocument.votingSummary.partnerDAO.votes")
+  public get partnerDAOVotes(): Array<IVoteInfo> {
+    return this.dealDocument.votingSummary.partnerDAO?.votes ?? [];
+  }
+
+  @computedFrom("allVotes")
+  public get submittedVotes(): Array<IVoteInfo> {
+    return this.allVotes.filter(voteInfo => voteInfo.vote === true || voteInfo.vote === false);
+  }
+
+  representativeVote(representativeAddress: string): boolean | null {
```

Comment:
> "representativeVoted" is misleading because the current account could be Proposal Lead but not a Rep. Should rename to "usedHasVoted"?


Based [on the SOT](https://www.notion.so/primedao/Partnered-Deal-Dashboard-79246b1b660445c0be3dca8fd4af5c43#9117b908ed2d48a0b2d65f5dc9bd57b4): "Proposal lead cannot vote and deposit/withdraw tokens, in order to do so proposal lead should also be registered as a representative."

A Proposal Lead needs to be a representative in order to vote. I will rename the method anyways.